yarn.lock

```diff
@@ -4,13 +4,13 @@
 
 bats-assert@ztombol/bats-assert:
   version "0.3.0"
-  resolved "https://codeload.github.com/ztombol/bats-assert/tar.gz/9f88b4207da750093baabc4e3f41bf68f0dd3630"
```

Comment:
Ugh, sorry. Will revert this. Not sure why running `yarn` did this. None of the hashes changed at least.